jdaviz/configs/specviz/plugins/viewers.py

```diff
@@ -545,8 +545,23 @@ def _plot_uncertainties(self):
                 self.figure.marks = list(self.figure.marks) + [error_line_mark]
 
     def set_plot_axes(self):
-        # Set axes labels for the spectrum viewer
+        # Set y axes labels for the spectrum viewer
+        y_display_unit = self.state.y_display_unit
+        y_unit = u.Unit(y_display_unit) if y_display_unit else u.dimensionless_unscaled
```

Comment:
Should `u.dimensionless_unscaled` be dumped into "Counts" category?

p.s. I thought I added this as part of the "review" but looks like it didn't go in.